src/EquationGenerator.java

```diff
@@ -0,0 +1,65 @@
+public class EquationGenerator {
+    String[] correctEquations;
+
+    void equationResult() {

```

Comment:
equationResult is not debesta name, I'd suggest generateEquations or generateCorrectEquations

Reply:
Done